sdk/storage/Azure.Storage.Common/src/Constants.cs

```diff
@@ -22,7 +22,7 @@ internal static class Constants
         /// Gets the default service version to use when building shared access
         /// signatures.
         /// </summary>
-        public const string DefaultSasVersion = "2018-11-09";
+        public const string DefaultSasVersion = "2019-02-02";
```

Comment:
That was it...?  Sigh.  Thanks for fixing this!

Reply:
Ya, and it actually broke Identity SAS.